paddle/operators/math/im2col.cc

```diff
@@ -127,19 +126,26 @@ class Col2ImFunctor<paddle::operators::math::ColFormat::kCFO,
     T* im_data = im->data<T>();
     const T* col_data = col.data<T>();
 
+    int w_offset = -1;
+    int h_offset = 0;
+    int c_im = 0;
     for (int c = 0; c < channels_col; ++c) {
-      int w_offset = c % filter_width;
-      int h_offset = (c / filter_width) % filter_height;
-      int c_im = c / filter_width / filter_height;
+      ++w_offset;
```

Comment:
Please leave a comment here. For speed.

Reply:
I found that this change has little impact on performance, and it will be reverted in the next commit.